TombEngine/Objects/TR5/Object/tr5_pushableblock.cpp

```diff
@@ -493,160 +508,145 @@ void PushableBlockCollision(short itemNumber, ItemInfo* laraItem, CollisionInfo*
 		PushableBlockBounds.BoundingBox.Z1 = bounds.Z1 - 200;
 		PushableBlockBounds.BoundingBox.Z2 = 0;
 
-		short rot = pushableItem->Pose.Orientation.y;
+		short yOrient = pushableItem->Pose.Orientation.y;
 		pushableItem->Pose.Orientation.y = (laraItem->Pose.Orientation.y + ANGLE(45.0f)) & 0xC000;
 
 		if (TestLaraPosition(PushableBlockBounds, pushableItem, laraItem))
 		{
-			unsigned short quadrant = (unsigned short)((pushableItem->Pose.Orientation.y / 0x4000) + ((rot + 0x2000) / 0x4000));
+			int quadrant = int((pushableItem->Pose.Orientation.y / 0x4000) + ((yOrient + 0x2000) / 0x4000));
```

Comment:
Why `GetQuadrant` is not used here? Or calculation is different, and just the name is the same?

Reply:
The calculation was a little different here so I left it alone, but I've figured it out now!